tests/UI/pages/BO/catalog/products/index.js

```diff
@@ -183,10 +183,16 @@ class Product extends BOBasePage {
    * Get Product Status
    * @param page
    * @param row
-   * @returns {Promise<string>}
+   * @returns {Promise<boolean>}
    */
   async getProductStatusFromList(page, row) {
-    return this.getTextContent(page, this.productsListTableColumnStatus(row));
+    const inputValue = await this.getAttributeContent(
+      page,
+      `${this.productsListTableColumnStatusInput(row)}[checked]`,
+      'value',
+    );
+
+    return inputValue !== '0';
```

Comment:
This was a string before and now it's a boolean, all the tests are fixed ?

Reply:
Only one tests used the function, and it's fixed